src/main/java/org/asciidoctor/asciidoclet/Stylesheets.java

```diff
@@ -84,10 +92,26 @@ private InputStream getResource(String name) throws IOException {
     }
 
     String selectStylesheet(String javaVersion) {
-        if (javaVersion.matches("^(11)(\\.)?.*")) {
-            return JAVA11_STYLESHEET;
+      String ret;
```

Comment:
This line indentation seems off. I suspect you have an IDE that indents to 2 spaces? The code is indented to 4. Not that is blocking, but I'd like to keep the code all aligned.
Same for `LazyDocCommentTableProcessorTest` seems to be indented to 2 spaces.